packages/api/internal/orchestrator/placement/placement.go

```diff
@@ -71,9 +75,13 @@ func PlaceSandbox(ctx context.Context, tracer trace.Tracer, algorithm Algorithm,
 		err = node.SandboxCreate(ctx, sbxRequest)
 		span.End()
 		if err != nil {
+			if algorithm.excludeNode(err) {
```

Comment:
for best of K we want to remove the check for in progress sandboxes from API, this will also help with tests / small number of nodes and bursts